.github/workflows/ci.yaml

```diff
@@ -55,6 +55,12 @@ jobs:
       - name: Test
         run: poetry run pytest tests
 
+      - name: Install docs dependencies
+        run: poetry install --with docs
```

Comment:
any reason for not adding this to the initial poetry install above?

Reply:
to save time if anything fails fefore that

Reply:
ah, yeah that makes sense. I don't have a preference, you can decide yourself what you think is most appropriate

Reply:
I prefer to keep it like it is currently, it seems that it is only installing the doc dependencies after all